contrib/mesos/pkg/executor/executor.go

```diff
@@ -208,23 +202,36 @@ func (k *KubernetesExecutor) isDone() bool {
 	}
 }
 
+func (k *KubernetesExecutor) SlaveInfo() *mesos.SlaveInfo {
+	return k.slaveInfo
```

Comment:
this is totally in line with how the rest of the framework abuses protobufs. I'd love for it to stop. my preference here would be to declare a struct in this package:
```go
type SlaveInfo struct {
  Hostname string
  Labels map[string]string
  Resources api.ResourceList
}
```
and build one of these in Registered, and return it from SlaveInfo(). This has the nice side-affect of stopping the protobuf bleeding into the cadvisor helper code.